card/lib/card/query.rb

```diff
@@ -53,8 +53,8 @@ module Query
                           refer_to referred_to_by
                           member_of member
 
-                          found_by not sort match name_match complete
-                          junction_complete extension_type],
```

Comment:
`name_match` and `junction_complete` seem like hacky add-ons.  We can already achieve `name_match` with `{ name: ["match", X] }`.  (it needs improvement, but we don't want to add new attributes to avoid fixing things).

junction_complete is a weirder (metric-specific) need but was being used as our default complete pattern.  It slows down normal completions and really isn't necessary, especially with my name_match simplifications.